packages/helpers/logger.ts

```diff
@@ -1,4 +1,4 @@
-type Log = (message: string) => void;
+type Log = typeof console.log;
```

Comment:
This change appears to tie the type to a specific implementation. We defined the type that way for adaptability to various implementations (e.g., file system, Slack, etc). Are there particular reasons for binding this type to the `console.log` implementation?

Reply:
Ah I see, I like the idea of extending behaviour, improving the DX of the logger 👍  Imo it's safer if you explicitly tell what use cases we intend to support instead of saying "whatever console.log supports". For example, newer versions of node.js might introduce new stuff into `console.log` thus changing the contract. If you think that's not a concern because we'll always use only the `console.log` implementation then it's fine 👍